### PR TITLE
ur mom

### PR DESCRIPTION
**Pull request by BryceWren**
did some preliminary work to set up registering mod items in a DiaperItem.java file

### CODE REVIEW

File: src/main/java/net/dungdealers/diapersplus/DiaperItem/DiaperItem.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import net.fabricmc.fabric.api.itemgroup.v1.FabricItemGroupEntries;
+import net.fabricmc.fabric.api.itemgroup.v1.ItemGroupEvents;
+import net.minecraft.item.Item;
+import net.dungdealers.diapersplus.Diapers;
+import net.minecraft.item.ItemGroups;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.util.Identifier;
+
+public class DiaperItem {
```
Comment (rowleynt) [inline src/main/java/net/dungdealers/diapersplus/DiaperItem/DiaperItem.java:13]: If this is going to be our main item class for the mod, we should give it a more general name like DPItems. A more specific class for the functionality of the diapers (I fucking typed this shit unironically) will be made later (watch episode 5 of the tutorial series for 'advanced items')

File: src/main/java/net/dungdealers/diapersplus/Diapers.java [hunk 2]
```diff
@@ -12,5 +13,7 @@ public class Diapers implements ModInitializer {
 	@Override
 	public void onInitialize() {
 		LOGGER.info("poop");
+		DiaperItem.registerModItems();
```
Comment (rowleynt) [inline src/main/java/net/dungdealers/diapersplus/Diapers.java:16]: make sure you change this to DPItems / whatever you change it to

File: src/main/java/net/dungdealers/diapersplus/DiaperItem/DiaperItem.java (hunk 1 above)
Comment (rowleynt) [inline src/main/java/net/dungdealers/diapersplus/DiaperItem/DiaperItem.java]: If this is going to be our main item class for the mod, we should give it a more general name like DPItems. A more specific class for the functionality of the diapers (I fucking typed this shit unironically) will be made later (watch episode 5 of the tutorial series for 'advanced items')

File: src/main/java/net/dungdealers/diapersplus/Diapers.java (hunk 2 above)
Comment (rowleynt) [inline src/main/java/net/dungdealers/diapersplus/Diapers.java]: make sure you change this to DPItems.registerModItems() / whatever you change it to

**Review by rowleynt — CHANGES_REQUESTED** [review]

**Review by rowleynt — APPROVED** [review]
thanks sexy

**Review by WarpedCakez — APPROVED** [review]
i trust nathan

**Pull request merged**